### PR TITLE
TN-10803 Add containerization

### PR DESCRIPTION
**Pull request by anr-tonic**
`helm template -f ./values.sample.yaml .` w/ and w/o containerization object in values.sample.yaml

### CODE REVIEW

File: templates/rbac-containerization.yaml [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{{- if ((.Values.containerization).rbac).create }}
```
Comment (anr-tonic) [inline templates/rbac-containerization.yaml:1]: Do we need an additional over all containerization on/off toggle? We didn't need one in our testing environment but is maybe convenient here.

File: values.sample.yaml [hunk 2]
```diff
@@ -140,6 +140,20 @@ tonicai:
...
+  #   - apiGroups: [""]
+  #     resources: ["pods"]
+  #     verbs: ["get", "list", "delete", "create", "watch", "deletecollection"]
+  #   - apiGroups: [""]
+  #     resources: ["secrets"]
+  #     verbs: ["list", "delete", "create", "deletecollection"]
+  rbac:
+    create: true
+  datapacker:
+    imageRepo: quay.io/tonicai/datapacker
```
Comment (anr-tonic) [inline values.sample.yaml:156]: Should this live under a key? A features key here or the one on tonic web? Keys are hard to rehome after landing in a public version.

File: templates/rbac-containerization.yaml (hunk 1 above)
Comment (calexander3) [reply templates/rbac-containerization.yaml:1]: The concern being we might not need these extra permissions?

Comment (anr-tonic) [reply templates/rbac-containerization.yaml:1]: I think I'm over thinking it tbh.

**Pull request merged**